extensions/wikia/ARecoveryEngine/ARecoveryUnlockCSS.class.php

```diff
@@ -5,7 +5,7 @@ class ARecoveryUnlockCSS {
 	const API_ENDPOINT = 'sp_create_csurl';
 	const CACHE_TTL = 3600 * 10; //10h
 	const TIMEOUT = 10;
-	const WIKIA_PROXY_ENDPOINT = '/__are';
+	const WIKIA_PROXY_ENDPOINT = '/__bre';
 
 	public static function getUnlockCSSUrl() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)